FunctionalTableData/CellStyle.swift

```diff
@@ -13,7 +13,22 @@ import UIKit
 ///
 /// Some properties are only supported by `UITableView`.
 public struct CellStyle {
-	public static var selectionColor: UIColor? = nil // TODO: make this not a static like this
+	/// The default selection color to use for styles that don't specify their own.
+	public static var defaultSelectionColor: UIColor? = nil
```

Comment:
I know we said "hey let's not do this", but I'm not averse to this approach; it's a pragmatic way of getting UIAppearance-style behaviour on a struct.

Reply:
oh I read that, and completely blanked after that.
🙆‍♂️